content/tutorials/quickstart/get-nftoken-list.md

```diff
@@ -0,0 +1,337 @@
+---
+html: get-nftoken-list.html
+parent: xrpl-quickstart.html
+blurb: List extant tokens on the XRP Ledger, get their data from the Clio server.
+labels:
+  - NFTokens
+  - Quickstart
+---
+
+# Get a List of NFTokens
+
+You can list all of the tokens that currently exist on the XRP Ledger. The process involves repeatedly using the `ledger_data` request, filtering the `NFToken` page objects, then listing the array of `NFToken` objects stored on the page. This example demonstrates how to extract `NFToken` objects and `NFTokenIDs`, and how to get information for the listed NFTokens using the `nft_info` request to the Clio server.
+
+## Usage
+
+You can download the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip){.github-code-download} archive to try the sample in your own browser.
+
+### Get NFTokens
+
+All transactions on the XRP Ledger are available for public review. You do not need an account or credentials to access information about minting and trading NFTokens.
+
+To get a list of current `NFTokens` on the XRP Ledger, click **Get NFTokens**. The button retrieves 200 transactions (the maximum batch allowance), then filters to show only the information from `NFToken` pages. The row marker for the response is stored in the **Marker** field. Each new search starts at the marker to return a unique set of records for the next 200 transactions. Click **Get NFTokens** repeatedly to retrieve all of the `NFTokens` on the XRP Ledger.
```

Comment:
I'd add a note here that actually retrieving _all_ the NFTokens is likely to a a _lot_ of calls and a very long time on Mainnet because it needs to iterate over the entire ledger contents (including non-NFT data), which is multiple gigabytes. And doing it too fast might get you rate-limited, so you might need to run your own server to do this in a timely fashion.